backend/app/services/helpOffers/getByRequestId.ts

```diff
@@ -0,0 +1,14 @@
+import * as helpOffersRepo from "../../repositories/helpOffers/getByRequestId";
+import { AppError } from "../../common/errors/AppError";
+import { errors } from "../../common/errors/errors";
+
+export const getByRequestId = async (help_request_id: number) => {
+    const helpOffersByRequestId = await helpOffersRepo.getByRequestId(help_request_id);
+    if (helpOffersByRequestId.length === 0) {
```

Comment:
It's not clear here whether help_request was not founf or the help_offer just doesn't exist